src/index.html

```diff
@@ -5,9 +5,69 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../src/styles/index.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-image-container">
+          <img
+            class="product-image"
+            src="../src/images/imac.jpeg"
+            alt="image"
+          >
+        </div>
+
+        <h2 class="product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="product-code">
+          Product code: 195434
+        </p>
+
+        <div class="rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="product-rate-quantity">
+            <p class="product-rate-quantity-text">
+              Reviews: 5
+            </p>
+          </div>
+        </div>
+
+        <div class="product-price">
+          <p class="product-price-text">
+            Price:
+          </p>
+          <p class="product-price-number">
+            $2,199
+          </p>
+        </div>
+
+        <button class="product-buy">
+          <a
+            href="#"
+            class="product-buy-link"
+            data-qa="hover"
+          >
```

Comment:
This HTML structure is not valid, consider using a link tag or button instead of mixing both